src/code_42e0_len_1f60.c

```diff
@@ -190,19 +298,46 @@ void add_vec2D_polar(f32* x, f32* y, f32 r, f32 theta) {
     *y -= r * cosTheta;
 }
 
-INCLUDE_ASM(s32, "code_42e0_len_1f60", _wrap_trig_lookup_value);
+u16 _wrap_trig_lookup_value(f32 theta) {
+    f32 ret = theta;
 
-INCLUDE_ASM(void, "code_42e0_len_1f60", sin_cos_rad, f32 rad, f32* outSinTheta, f32* outCosTheta);
+    if (theta >= 1048576.0f || theta <= -1048576.0f) {
+        ret = theta * 0.0000009536743f;
+        ret = theta - ((s32) ret) * 1048576.0f;
+    }
 
-INCLUDE_ASM(f32, "code_42e0_len_1f60", sin_rad, f32 angle);
+    return (s32) ret;
+}
 
-INCLUDE_ASM(f32, "code_42e0_len_1f60", cos_rad, f32 angle);
+void sin_cos_rad(f32 rad, f32* outSinTheta, f32* outCosTheta) {
+    u16 theta = _wrap_trig_lookup_value(rad * 10430.378f);
 
-INCLUDE_ASM(s32, "code_42e0_len_1f60", sin_cos_deg);
+    *outSinTheta = sins(theta) * 3.051851e-05;
+    *outCosTheta = coss(theta) * 3.051851e-05;
```

Comment:
should definitely #define whatever this is

Reply:
I think it might be better to wait until we can give it a proper name before we define it